src/Microsoft.AspNetCore.Blazor/Components/BindMethods.cs

```diff
@@ -35,10 +35,10 @@ public static string GetEventHandlerValue<T>(string value)
         /// <summary>
         /// Not intended to be used directly.
         /// </summary>
-        public static UIEventHandler GetEventHandlerValue<T>(Action<T> value)
+        public static MulticastDelegate GetEventHandlerValue<T>(Action<T> value)
             where T : UIEventArgs
         {
-            return e => value((T)e);
+            return value;
```

Comment:
This is the payoff. This will now 'just work', and doesn't cause a diff each time the component renders (except in the case of a capturing lambda).

The advantage of having the tag helper at all (vs no language support) is that because we know what `T` is at compile time, is that you get a much better experience using a lambda. Without the language support, lambdas would have to be `UIEventHandler` **only**

Reply:
Excellent!